src/librbd/cache/ImageWriteback.h

```diff
@@ -33,6 +37,7 @@ class ImageWriteback {
   void aio_discard(uint64_t offset, uint64_t length,
                    uint32_t discard_granularity_bytes, Context *on_finish);
   void aio_flush(Context *on_finish);
```

Comment:
Nit: definitely want to avoid this once the classes are collapse (two `aio_flush` methods and one has a source).

Reply:
Done